test/parallel/test-process-cpuUsage.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+
+const result = process.cpuUsage();
+
+// Validate the result of calling with no previous value argument.
+validateResult(result);
+
+// Validate the result of calling with a previous value argument.
+validateResult(process.cpuUsage(result));
+
+// Ensure the results are monotonically increasing.
+let thisUsage;
+let lastUsage = process.cpuUsage();
+for (let i = 0; i < 10; i++) {
+  thisUsage = process.cpuUsage();
+  assert(thisUsage.user >= lastUsage.user);
+  assert(thisUsage.system >= lastUsage.system);
+  lastUsage = thisUsage;
+}
+
+// Ensure that the diffs are greater than 0.
+lastUsage = process.cpuUsage();
+for (let i = 0; i < 10; i++) {
+  thisUsage = process.cpuUsage(lastUsage);
+  assert(thisUsage.user >= 0);
+  assert(thisUsage.system >= 0);
```

Comment:
comment doesn't match test. "Ensure that the diffs are greater than 0" != checking `>=`. Should just be `> 0`.